Source/NomadContent.cpp

```
@@ -524,36 +919,205 @@ void NomadContent::syncViewState() {
 }
 
 void NomadContent::setViewFocus(ViewFocus focus) {
```

Clang (14.0.6)

[warning] function 'setViewFocus' has cognitive complexity of 52 (threshold 25)

(readability-function-cognitive-complexity)